models/User.ts

```diff
@@ -0,0 +1,68 @@
+import { model, Schema } from "mongoose";
+import IUser from "../types/IUser";
+
+const UserSchema = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  givenName: {
+    type: String,
+    required: true,
+  },
+  familyName: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+  },
+  profilePicture: {
+    type: String,
```

Comment:
Maybe comment how a string will relate to a profile picture for ease of understanding of how the code will work.

Reply:
The string is actually just the src url of the image, so creating an image with `<img src={profilePicture} />` would point to the image hosted on Google's server for us